pkg/detectors/maxmindlicense/v2/maxmindlicense_v2.go

```diff
@@ -37,49 +36,43 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	dataStr := string(data)
 
 	keyMatches := keyPat.FindAllStringSubmatch(dataStr, -1)
-	idMatches := idPat.FindAllStringSubmatch(dataStr, -1)
 
 	for _, keyMatch := range keyMatches {
 		keyRes := strings.TrimSpace(keyMatch[1])
 
-		for _, idMatch := range idMatches {
-			if len(idMatch) != 2 {
+		r := detectors.Result{
+			DetectorType: detectorspb.DetectorType_MaxMindLicense,
+			Redacted:     keyRes,
+			Raw:          []byte(keyRes),
+			ExtraData: map[string]string{
+				"rotation_guide": "https://howtorotate.com/docs/tutorials/maxmind/",
+				"version":        fmt.Sprintf("%d", s.Version()),
+			},
+		}
+
+		if verify {
+			req, err := http.NewRequestWithContext(ctx, "POST", "https://secret-scanning.maxmind.com/secrets/validate-license-key", nil)
+			if err != nil {
 				continue
 			}
-			idRes := strings.TrimSpace(idMatch[1])
-
-			s := detectors.Result{
-				DetectorType: detectorspb.DetectorType_MaxMindLicense,
-				Redacted:     idRes,
-				Raw:          []byte(keyRes),
-				RawV2:        []byte(keyRes + idRes),
-				ExtraData: map[string]string{
-					"rotation_guide": "https://howtorotate.com/docs/tutorials/maxmind/",
-					"version":        fmt.Sprintf("%d", s.Version()),
-				},
-			}
 
-			if verify {
-				req, err := http.NewRequestWithContext(ctx, "GET", "https://geoip.maxmind.com/geoip/v2.1/country/8.8.8.8", nil)
-				if err != nil {
-					continue
-				}
-				req.SetBasicAuth(idRes, keyRes)
-				res, err := client.Do(req)
-				if err == nil {
-					defer res.Body.Close()
-					if res.StatusCode >= 200 && res.StatusCode < 300 {
-						s.Verified = true
-					} else {
-						if detectors.IsKnownFalsePositive(keyRes, detectors.DefaultFalsePositives, true) {
-							continue
-						}
+			req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+			req.Form.Add("license_key", keyRes)
+
+			res, err := client.Do(req)
+			if err == nil {
+				defer res.Body.Close()
+				if res.StatusCode >= 200 && res.StatusCode < 300 {
+					r.Verified = true
+				} else {
```

Comment:
I'm not sure this `IsKnownFalsePositive` branch is needed. It doesn't do anything when it does detect a false positive, and it looks like the values for `detectors.DefaultFalsePositives` wouldn't match our regex anyway